src/test/scala/ChiselSpec.scala

```diff
@@ -22,40 +23,27 @@ import java.io.{ByteArrayOutputStream, PrintStream}
 import scala.reflect.ClassTag
 
 /** Common utility functions for Chisel unit tests. */
-trait ChiselRunners extends Assertions {
-  def runTester(
-    t:                    => BasicTester,
-    additionalVResources: Seq[String] = Seq()
-//                 annotations:          AnnotationSeq = Seq()
-  ): Boolean = {
-    val defaultBackend = chisel3.testers.TesterDriver.defaultBackend
-//    val hasBackend = TestUtils.containsBackend(annotations)
-//    val annos: Seq[Annotation] = if (hasBackend) annotations else defaultBackend +: annotations
-    TesterDriver.execute(() => t, additionalVResources)
+trait ChiselRunners extends TestSuite with Assertions with ChiselSim {
+  def runTester(t: => Module, maxCycles: Int = 1000): Unit = {
+    simulate(t)(RunUntilFinished(maxCycles))
   }
```

Comment:
Consider for a follow-on (or in this PR). The `runTester` method is so close to doing nothing at this point, that you may want to remove it and inline the `simulate` methods directly. This "ChiselRunners" pattern was started in Chisel and got exported a lot of places. However, I really prefer direct usage of the `ChiselSim` at this point for clarity.

Reply:
Addressed in https://github.com/ucb-bar/fixedpoint/pull/11/commits/28d70785628c5784c10a280c926e7d02549c4652.